netbox/ipam/models/ip.py

```diff
@@ -125,11 +125,22 @@ class Meta:
         verbose_name_plural = 'ASNs'
 
     def __str__(self):
-        return f'AS{self.asn}'
+        if self.asn > 65535:
+            return 'AS{} ({}.{})'.format(self.asn, self.asn // 65536, self.asn % 65536)
```

Comment:
This should probably return `self.asdot_notation`.

Reply:
I think we should also stick with fstrings if we can IMO

Reply:
Updated to use fstring and to return `asn_with_asdot` per below suggestion. Resulting code is much cleaner.